Include/httpClient/httpClient.h

```diff
@@ -446,7 +446,7 @@ STDAPI HCHttpCallRequestSetRetryAllowed(
 /// ID number of this REST endpoint used to cache the Retry-After header for fast fail.
 /// </summary>
 /// <param name="call">The handle of the HTTP call.  Pass nullptr to set the default for future calls.</param>
-/// <param name="retryAfterCacheId">ID number of this REST endpoint used to cache the Retry-After header for fast fail.  1-1000 are reserved for XSAPI.</param>
+/// <param name="retryAfterCacheId">ID number of this REST endpoint used to cache the Retry-After header for fast fail. Must be non-0. 1-1000 are reserved for XSAPI.</param>
```

Comment:
nit: elsewhere these docs spell out zero as a word rather than a number. recommend "non-zero" for consistency (also I think it reads better)